tests/units/typing/url/test_text_url.py

```diff
@@ -8,7 +8,7 @@
 from docarray.typing import TextUrl
 from tests import TOYDATA_DIR
 
-REMOTE_TEXT_FILE = 'https://de.wikipedia.org/wiki/Brixen'
+REMOTE_TEXT_FILE = 'https://www.gutenberg.org/files/1065/1065-0.txt'
```

Comment:
is this test needed? I do not like to have a test depemding on an external website that can remove the image

Reply:
but we were already doing that with wikipedia

Reply:
the purpose is to test loading remote text files, I think gutenberg project is very reliable, if not we can easily remove it